system/security/LdapSecurity/ldaputils.cpp

```diff
@@ -68,19 +68,27 @@ LDAP* LdapUtils::LdapInit(const char* protocol, const char* host, int port, int
 #ifndef HAVE_TLS
         //throw MakeStringException(-1, "openldap client library libldap not compiled with TLS support");
 #endif
+        int rc;
```

Comment:
@RussWhitehead can you add a check to log the cipherSuite used if it is not empty ?